README.md

```diff
@@ -1,39 +1,43 @@
-# Coffee Tasks
+# Coffee Tasks :coffee:
```

Comment:
You can remove this coffee emoji.
It's not in sync with the app's logo.

Reply:
@SauveJeanLuc done :)